### ISSUE TITLE
Failed to finetune ResNet50 on UCF101 split-1

### ISSUE THREAD

**Issue by rogercmq** [okankop/Efficient-3DCNNs#33]
Hi! There are some implementation details on training 3DCNNs on UCF101 in your paper[1], one of which is "While dropout ratio is kept at 0.2 for Kinetics600 and Jester, it is increased to 0.9 for UCF-101". However, I cannot see any dropout modules in your ``resnet.py``. 

So far, I haven't produced the results (88.92% in [Tab.8](https://github.com/okankop/Efficient-3DCNNs/blob/master/utils/results.jpg)) in your paper. Could you give me an example run of resnet50-ucf101-pretrainK600?

Besides, what is your codebase for I3D?

Regards. 

[1] 《Resource Efficient 3D Convolutional Neural Networks》
